vuepress/docs/docs/adapters.md

```diff
@@ -3,9 +3,16 @@
 Once you have an interpreter able to execute GraphQL queries, you usually want to expose it using an HTTP API.
 Caliban comes with a few "ready-to-use" components (called "adapters") to expose your API with the most popular HTTP libraries.
 
+::: tip `QuickAdapter`
+Starting with v2.4.3, Caliban provides the opinionated [QuickAdapter](adapters.md#high-performance-quickadapter)
+that favours ease-of-use and performance at the expense of customizability.
+
+If you don't mind using `zio-http` as the server implementation, make sure to check it out!
```

Comment:
How about instead "If you want the best possible performance, make sure to check it out!"